### PR TITLE
Thomas/mar 534 create UI for a data model pivot

### PR DESCRIPTION
**Pull request by balzdur**
Small update did during the create pivot implementation described [here](https://github.com/checkmarble/marble-frontend/pull/435)

### CODE REVIEW

File: dto/data_model_dto.go [hunk 1]
```diff
@@ -106,6 +107,7 @@ func adaptDataModelField(field models.Field) Field {
 
 func adaptDataModelLink(linkToSingle models.LinkToSingle) LinkToSingle {
 	return LinkToSingle{
+		Id:                     linkToSingle.Id,
```
Comment (balzdur) [inline dto/data_model_dto.go:110]: The main thing: I need link ids for pivot creation

File: api/routes.go [hunk 2]
```diff
@@ -144,8 +144,8 @@ func (api *API) routes(auth *Authentication, tokenHandler *TokenHandler) {
 	router.PATCH("/data-model/fields/:fieldID", api.UpdateDataModelField)
 	router.DELETE("/data-model", api.DeleteDataModel)
 	router.GET("/data-model/openapi", api.OpenAPI)
-	router.POST("data-model/pivots", api.createDataModelPivot)
-	router.GET("data-model/pivots", api.listDataModelPivots)
+	router.POST("/data-model/pivots", api.createDataModelPivot)
```
Comment (Pascal-Delange) [inline api/routes.go:147]: oh, did that make a difference ? 🤔 
It was working on postman at least

File: dto/data_model_dto.go (hunk 1 above)
Comment (Pascal-Delange) [reply dto/data_model_dto.go:110]: my bad, I thought I added this

**Review by Pascal-Delange — APPROVED** [review]

File: api/routes.go (hunk 2 above)
Comment (balzdur) [reply api/routes.go:147]: I think not, it was just uniformisation

Comment (Pascal-Delange) [reply api/routes.go:147]: ah great, thanks :)

**Pull request merged**